packages/@orchidui-vue/src/data/JsonForm.sample.js

```diff
@@ -114,13 +123,9 @@ const SampleJsonForm = [
       placeholder: "placeholder",
     },
   },
-  {
-    name: "toggle_field",
-    type: "Toggle",
-    className: "md:col-span-2",
-  },
   {
     name: "radio_group",
+    key: "radio_group",
```

Comment:
Can we use the name as the key.

Reply:
I think we could but what we do if name was an array?

<img width="311" alt="Screenshot 2023-11-20 at 10 51 24" src="https://github.com/hit-pay/orchid/assets/48382527/82bbe8af-9ab8-4312-b4f8-9bf3a7ac98c9">

Reply:
https://github.com/hit-pay/orchid/pull/80/commits/5e1829f0dc46ef16f9c7272c1275011829d195b9  @mykhailo-diakovych What if it's like this? Is this good or bad code.

Reply:
> [5e1829f](https://github.com/hit-pay/orchid/commit/5e1829f0dc46ef16f9c7272c1275011829d195b9) @mykhailo-diakovych What if it's like this? Is this good or bad code.

Yes, it is fine

Reply:
i will merge this PR, good idea for complex form builder